### ISSUE TITLE
Decorative icons should be hidden from screen readers

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#1097]
Add `aria-hidden="true"` to decorative icons (e.g., formats) so screen readers do not see them.
